demo-default.env

```diff
@@ -51,3 +50,42 @@ IGL_TEST_DBNAME=postgres_test
 # turn on unsafe testing repo methods
 IGL_ALLOW_UNSAFE_REPO_IS_EMPTY=True
 IGL_ALLOW_UNSAFE_REPO_CLEAR=True
+
+
+IGL_MCHR_ROUTING_TABLE=[{ "Name": "shared db channel to Australia", "Jurisdiction": "AU", "Predicate": "UN.CEFACT.", "ChannelUrl": "https://sharedchannel.services.devnet.trustbridge.io/", "ChannelAuth": "Cognito/JWT" }, { "Name": "shared db channel to Singapore", "Jurisdiction": "SG", "Predicate": "UN.CEFACT.", "ChannelUrl": "https://sharedchannel.services.devnet.trustbridge.io/", "ChannelAuth": "Cognito/JWT" }, { "Name": "Local FR channel", "Jurisdiction": "FR", "Predicate": "UN.CEFACT.", "ChannelUrl": "http://docker-host:7500/", "ChannelAuth": "None" }]
```

Comment:
Note: @adamkerz; at some point some of this will need to become dynamic (call channel API to find out what predicates/jurisdictions the channel supports). I *think* the only static/configured part of the routing table should be the precedence-ordered list of channels.

Reply:
yes, I think urls + channel type (always channel API so far but who knows) and auth variables will be the best choice

Reply:
Of course, @monkeypants, I've already noted that the routing table will be changing to db table of some form as soon as we have it working.